projects/aca-shared/src/lib/components/document-base-page/document-base-page.spec.ts

```diff
@@ -278,6 +278,22 @@ describe('PageComponent', () => {
       expect(searchAiService.updateSearchAiInputState).not.toHaveBeenCalled();
     });
   });
+
+  describe('SearchAiService toggleSearchAiInput$ event handler', () => {
+    it('should set searchAiInputState', () => {
+      const initialSearchAiInputState = component.searchAiInputState;
+      const searchAiInputState: SearchAiInputState = {
+        active: true
+      };
+      TestBed.inject(SearchAiService).toggleSearchAiInput$ = of(searchAiInputState);
```

Comment:
isn;'t it supposed to be:
searchAiService.toggleSearchAiInput$ = of(searchAiInputState);
?

Reply:
done